tests/Cryptor.unit.test.ts

```diff
@@ -1,124 +1,139 @@
 import Crypto from '../src/Crypto'
-import { Buffer } from 'buffer/'
+import * as nodeCryptoLib from 'crypto'
```

Comment:
Why not just `import * as crypto from 'crypto'`?

Reply:
this was to distinguish from our own `let crypto: Crypto` statements, we could rename either and I don't think it would matter

Reply:
So we can leave `crypto` to mean Node's crypto. As that's recognizable easily. But then our specific form of crypto can be prefixed as `EncryptedFSCrypto`.

Reply:
Done